sass/_10.download.scss

```diff
@@ -0,0 +1,46 @@
+.download {
+	@include section-padding();
+	background-color: var(--color-blue);
+	color: var(--color-white);
+	text-align: center;
+
+	&__title {
+		font-size: calc(0.5rem + 0.8vw);
+		font-weight: 700;
+		margin-bottom: 1rem;
+	}
+
+	&__para {
+		font-size: calc(0.4rem + 0.4vw);
+		margin-bottom: 1rem;
+	}
+
+	&__form {
+		& input {
```

Comment:
Skoro używamy BEM to powinniśmy używać klas do stylowania (wyjątej reset.css).
Wtedy też nie robiłbym dodatkowych zagnieżdżeń - BEM to tylko klasy czyli specyficzność na poziomie  (0, 1, 0)